test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env rspec
```

Comment:
I just follow the convention from Rails world :) I try to reproduce the same file structures on `src` and `test` directories. I moved it to test/include/ask_test.rb.